src/components/AutoSuggestion/AutoSuggestionList.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import computeScrollIntoView from "compute-scroll-into-view";
+import { compute } from "compute-scroll-into-view";
```

Comment:
Naming the function just `compute` is too generic when used somewhere in the code.

Reply:
They changed the exports. This is the way to use it, see https://www.npmjs.com/package/compute-scroll-into-view?activeTab=readme#usage